guides/common/assembly_using-pxe-to-provision-hosts.adoc

```diff
@@ -11,5 +11,114 @@ include::modules/proc_creating-hosts-with-pxeless-provisioning.adoc[leveloffset=
 
 include::modules/proc_creating-hosts-with-uefi-http-boot-provisioning.adoc[leveloffset=+1]
 
+ifndef::satellite[]
```

Comment:
Would you mind pulling the attributes into a standalone attributes file and including the file? For example:
```
include::common/attributes-secureboot.adoc[]
```